named_type_impl.hpp

```diff
@@ -31,8 +31,8 @@ class FLUENT_EBCO NamedType : public Skills<NamedType<T, Parameter, Skills...>>.
     explicit constexpr NamedType(T&& value) : value_(std::move(value)) noexcept(noexcept(T(std::move(value)))) {}
 
     // get
-    constexpr T& get() noexcept { return value_; }
-    constexpr std::remove_reference_t<T> const& get() const noexcept {return value_; }
+    [[nodiscard]] constexpr T& get() noexcept { return value_; }
```

Comment:
I agree in principle, but the lib is compatible with C++14.
I'll accept the PR and remove the attribute afterwards

Reply:
Makes sense. You could add a macro like `FLUENT_NODISCARD` or something I don't think its that important